src/main/java/uk/gov/ida/verifyserviceprovider/services/EidasUnsignedAssertionTranslator.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.ida.verifyserviceprovider.services;
+
+import org.opensaml.saml.saml2.core.Assertion;
+import uk.gov.ida.saml.core.transformers.MatchingDatasetUnmarshaller;
+import uk.gov.ida.saml.metadata.EidasMetadataResolverRepository;
+import uk.gov.ida.verifyserviceprovider.factories.saml.UserIdHashFactory;
+import uk.gov.ida.verifyserviceprovider.mappers.MatchingDatasetToNonMatchingAttributesMapper;
+import uk.gov.ida.verifyserviceprovider.validators.ConditionsValidator;
+import uk.gov.ida.verifyserviceprovider.validators.InstantValidator;
+import uk.gov.ida.verifyserviceprovider.validators.LevelOfAssuranceValidator;
+import uk.gov.ida.verifyserviceprovider.validators.SubjectValidator;
+
+import java.util.List;
+
+public class EidasUnsignedAssertionTranslator extends BaseEidasAssertionTranslator {
```

Comment:
In no way a blocking comment, but maybe in future consider composition for this sort of thing - have one EidasAssertionTranslator and two classes that implement the signature validation (interface) which are injected at creation (or get all modern and funky and use lambdas although that wouldn't give the testability that separate classes would).  Clearly one implementation would still be a no-op implementation.

Reply:
Interesting. Might be cleaner, I'll think about it next time.